### ISSUE TITLE
Explicit Table schema

### ISSUE THREAD

**Issue by isaacabraham** [fsprojects/AzureStorageTypeProvider#98]
Add the ability to explicitly identify a table schema through a static file.

**Comment by isaacabraham** [comment]
@mathias-brandewinder what do you think of a schema like this: -

```json
{
    "tables": [
        {
            "table": "myTable",
            "columns": [
                {
                    "Column": "columnA",
                    "Type": "String",
                    "Optional": true
                },
                {
                    "Column": "columnB",
                    "Type": "DateTime",
                    "Optional": true
                },
                {
                    "Column": "columnC",
                    "Type": "Int64"
                }
            ]
        }
    ]
}```

**Comment by isaacabraham** [comment]
@mathias-brandewinder would you be interested in testing this out? If you check out the latest commit in the `static-table-schema` branch, run `build.cmd` and then copy over the AzureStorageTypeProvider dll (in the bin folder) to wherever you're referencing the current version, you should be able to pass in another parameter, `tableSchema`, which points to the file that must look as per the above. This *appears* to work for me locally.

**Comment by mathias-brandewinder** [comment]
That looks reasonable. Can't promise how quickly I can test this out, but I'll try my best - at the very least, I'll give the approach some thought :)

**Comment by isaacabraham** [comment]
Fixed by #99 .
